winston-logger/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "winston-logger",
+  "version": "1.0.0",
+  "description": "example how to use winston logger with Fastify v2",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "standard"
```

Comment:
could you add also a script that execute curl? 
`try-me: "curl http://localhost:3000/hello"`